websites/D/Discohook/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "Discohook",
+  "author": {
+    "id": "150987570715688960",
+    "name": "Ninja#4321"
+  },
+  "description": {
+    "en": "Discohook is a free tool that allows you to personalise your server to make your server stand out from the crowd."
+  },
+  "url": "discohook.org",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/mQ5aH7h.png",
```

Comment:
Logos should be at least 512x512 pixels. Find/create a larger logo. You can also use an upscaling tool like waifu2x

Reply:
Logo updated with 512x512 version